interceptors/GlobalData.cfc

```diff
@@ -20,7 +20,7 @@ component {
 			"expandAll" : prc.expandedResourceDivs,
 			"sessionEnabled" : getInstance( "APIService@relax" ).getSessionsEnabled(),
 			"url" : {
-				"moduleRoot" : replacenocase( event.buildLink( linkTo='relax', ssl=event.isSSL() ), "/index.cfm", "" ),
+				"moduleRoot" : replacenocase( event.buildLink( to='relax', ssl=event.isSSL() ), "/index.cfm", "" ),
```

Comment:
Let's just make this backwards compatbile @homestar9 just replace with 
`event.buildLink( 'relax' )`, since v5 we don't need the ssl check there.

Reply:
Sounds good!